### ISSUE TITLE
GNOME 42: prefsModule.buildPrefsWidget is not a function

### ISSUE THREAD

**Issue by hedw1gP** [aunetx/blur-my-shell#219]
The settings of extension blur-my-shell@aunetx had an error:
```
TypeError: prefsModule.buildPrefsWidget is not a function
```

Stack trace:
```
_init@resource:///org/gnome/Shell/Extensions/js/extensionsService.js:219:40
ExtensionPrefsDialog@resource:///org/gnome/Shell/Extensions/js/extensionsService.js:175:4
OpenExtensionPrefsAsync/<@resource:///org/gnome/Shell/Extensions/js/extensionsService.js:128:33
asyncCallback@resource:///org/gnome/gjs/modules/core/overrides/Gio.js:115:22
run@resource:///org/gnome/Shell/Extensions/js/dbusService.js:186:20
main@resource:///org/gnome/Shell/Extensions/js/main.js:19:13
run@resource:///org/gnome/gjs/modules/script/package.js:206:19
start@resource:///org/gnome/gjs/modules/script/package.js:190:8
@/usr/share/gnome-shell/org.gnome.Shell.Extensions:1:17
```

My neofetch: 

```
hedw1g@pop-os 
------------- 
OS: Pop!_OS 22.04 LTS x86_64 
Host: ROG Zephyrus G14 GA401IV_GA401IV 1.0 
Kernel: 5.16.15-76051615-generic 
Uptime: 12 hours, 18 mins 
Packages: 2824 (dpkg), 4 (flatpak), 5 (snap) 
Shell: bash 5.1.16 
Resolution: 2560x1440 
DE: GNOME 42.0 
WM: Mutter 
WM Theme: WhiteSur-dark 
Theme: WhiteSur-dark [GTK2/3] 
Icons: WhiteSur-dark [GTK2/3] 
Terminal: gnome-terminal 
CPU: AMD Ryzen 7 4800HS with Radeon Graphics (16) @ 2.900GHz 
GPU: NVIDIA GeForce RTX 2060 Max-Q 
GPU: AMD ATI 04:00.0 Renoir 
Memory: 4168MiB / 15478MiB 
```

**Comment by aunetx** [comment]
Hello, which version of the extension do you use? And how did you install gnome 42?

It seems either you have an ancient version of blur my shell, or the gnome version you installed does not contains the changes necessary (meaning, you have a pre-beta build)

**Comment by hedw1gP** [comment]
It is the current version from extensions.gnome.org;

It is a beta build from pop-os/beta, I just upgraded from 20.10 with `pop-upgrade`; but in the last couple of hours it just destroyed itself, apt autoremoved some important stuff. So I guess it’s Pop’s fault.

I reinstalled my computer to Arch Linux and no issues. I’m closing this since it is probably not your fault, and I was using a beta unstable version.
